tests/contrib/aiohttp/test_aiohttp_client.py

```diff
@@ -0,0 +1,177 @@
+import os
+
+import aiohttp
+import pytest
+
+from ddtrace.contrib.aiohttp import patch
+from ddtrace.contrib.aiohttp import unpatch
+from tests.utils import override_http_config
+
+from ..config import HTTPBIN_CONFIG
+
+
+HOST = HTTPBIN_CONFIG["host"]
+PORT = HTTPBIN_CONFIG["port"]
+SOCKET = "{}:{}".format(HOST, PORT)
+URL_200 = "http://{}/status/200".format(SOCKET)
+URL_500 = "http://{}/status/500".format(SOCKET)
+
+
+@pytest.fixture(autouse=True)
+def patch_aiohttp_client():
+    patch()
+    yield
+    unpatch()
+
+
+@pytest.mark.asyncio
+async def test_200_request(snapshot_context):
+    with snapshot_context():
+        async with aiohttp.ClientSession() as session:
+            async with session.get(URL_200) as resp:
+                assert resp.status == 200
+
+
+@pytest.mark.asyncio
+async def test_500_request(snapshot_context):
+    with snapshot_context():
+        async with aiohttp.ClientSession() as session:
+            async with session.get(URL_500) as resp:
+                assert resp.status == 500
+
+
+@pytest.mark.asyncio
+async def test_200_request_distributed_tracing(tracer, test_spans):
+    class MockRequest(object):
```

Comment:
same here, why not httpbin?

Reply:
HTTPretty? which we already use elsewhere

Reply:
ah yes I forgot about HTTPretty!

Reply:
ah, makes sense, especially since we are on such an old version for Python 2 support.